libs/auth-api-lib/seeders/20201128002158-add-redirect-uri-for-dev.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface) => {
+    const redirectUri = [
+      {
+        client_id: 'island-is-1',
+        redirect_uri: 'https://localhost:5001/oauth2-redirect.html',
+      },
+    ]
+
+    if (process.env.NODE_ENV === 'development') {
```

Comment:
Don't use this variable. It had a special meaning. Use something that describes the feature please - SUPPORT_LOCALHOST_AUTH or something like that.

Reply:
Changed to SUPPORT_LOCALHOST_AUTH